src/Turbo/DependencyInjection/TurboExtension.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\DependencyInjection;
+
+use Doctrine\Bundle\DoctrineBundle\DoctrineBundle;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\MercureBundle\MercureBundle;
+use Symfony\Bundle\TwigBundle\TwigBundle;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+/**
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * @experimental
+ */
+final class TurboExtension extends Extension
+{
+    /**
+     * @param array<string, array> $configs
+     */
+    public function load(array $configs, ContainerBuilder $container): void
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = (new PhpFileLoader($container, new FileLocator(__DIR__.'/../Resources/config')));
+        $loader->load('services.php');
+
+        if ($config['broadcast']['enabled'] && 80000 > \PHP_VERSION_ID) {
+            throw new InvalidConfigurationException('Enabling the "broadcast" config option requires PHP 8 or higher.');
+        }
+
+        if (class_exists(TwigBundle::class)) {
+            $loader->load('twig.php');
+            if (isset($config['mercure']['subscribe_url'])) {
+                $container->getDefinition('turbo.twig.extension.stream')->addArgument($config['mercure']['subscribe_url']);
+            }
+
+            if (class_exists(MercureBundle::class)) {
+                $loader->load('broadcaster.php');
```

Comment:
Shouldn't we check for Doctrine presence as well? I'm not sure broadcasting make sense without Doctrine?

Reply:
It does. I adapted this code from API Platform, which already supports a lot of persistence systems. We could add more later (and users can add more in third party packages).

Reply:
Ah, nice, thanks :) !